src/main/scala/de/hpi/callcenterdashboard/DataStore.scala

```diff
@@ -362,193 +509,274 @@ class DataStore(credentials: CredentialsTrait) {
     products
   }
 
-  def getOutstandingOrdersOfCustomerUpTo(customer: Customer, date: FormattedDate): List[Order] = {
-    var orders = List.empty[Order]
+  /**
+    * Get the world wide sales sum in a given date range.
+    *
+    * @param filter The filter to apply.
+    * @return Triples of short country code, sales sum, country name.
+    */
+  def getWorldWideSales(filter: Filter): List[(Country, Money, List[(Region, Money)])] = {
```

Comment:
Monsterquery, aber sehr nice, dass du das in einer umgesetzt bekommen hast! Respekt!!